### PR TITLE
Add OL 21.0.0.5 GA Release blog to staging

### PR DESCRIPTION
**Pull request by austin0**
https://github.com/OpenLiberty/blogs/issues/1401

image

### CODE REVIEW

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (mbroz2) [inline posts/2021-05-14-ldap-multipart-21005.adoc]: this link is wrong.  Make sure to always try all links in the blog once it's on the draft/staging site to make sure they all work.

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (mbroz2) [inline posts/2021-05-14-ldap-multipart-21005.adoc]: @utle this looks incorrect; I'm guessing the `keytab` key needs to be removed (or otherwise provided a value)

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (mbroz2) [inline posts/2021-05-14-ldap-multipart-21005.adoc]: This is no longer beta.
"To enable this function, add the LDAP User Registry 3.0 feature to your `server.xml` file:"

**Review by mbroz2 — CHANGES_REQUESTED** [review]
Swap the order of everything (content, titles, ToC/navigation) to lead with JAX-RS, then Kerberos (keeping java se last)

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (austin0) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: My bad, used an old code snippet. I believe it should be:

```
<kerberos keytab="${server.config.dir}/security/krb5.keytab" configFile="${server.config.dir}/security/krb5.conf" />
```

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (austin0) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: ~~Removed as lacking significant bugs content.~~

Fixing while working to add significant bugs.

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (austin0) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: Updated.

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (mbroz2) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: @utle plz confirm the code snippet, whether the keytab should be there and if so that the value is correct

Comment (utle) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: It's looked good with Austin updated

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (austin0) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: Significant bugs have been added, 3/4 have been reviewed currently waiting on @nmittles for a final review.

Building on draft will update PR image when live.

Comment (austin0) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: All bugs reviewed for accuracy, rebuilding on draft.

Comment (mbroz2) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: the above link is still incorrect

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (mbroz2) [inline posts/2021-05-14-ldap-multipart-21005.adoc]: "this method should **return** a"

**Review by mbroz2 — CHANGES_REQUESTED** [review]

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (austin0) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: Corrected.

File: posts/2021-05-14-ldap-multipart-21005.adoc
Comment (austin0) [reply posts/2021-05-14-ldap-multipart-21005.adoc]: Link corrected.

**Review by mbroz2 — APPROVED** [review]

**Pull request merged**
